### PR TITLE
feature: multilanguage / multisite support

### PR DESCRIPTION
**Pull request by dkarlovi**
This PR will allow having multi-language / multi-site support, see #33.

The first step is adding configuration for both Elasticsearch and the sites, like so

```yaml
core_shop2_vue_storefront:
    elasticsearch:
        hosts: "%env(STOREFRONT_ELASTCSEARCH_DSN)%"
        index: "%env(STOREFRONT_ELASTCSEARCH_INDEX)%_{store}_{language}_{entity}"
    stores:
        example.nl:
            languages:
                - nl
         example.co.uk:
            languages:
                - en
        example.com:
            languages:
                - en
```

The tokens in the index name allow us to automagically separate per store / language / entity (Product, Category, etc), allowing much easier implementation for #30 also. Idea is to also auto-configure the ONGR bundle, so the ES configuration there will no longer be required.

The command now looks like `bin/console vsbridge:index example.co.uk en category` (any param is optional, the more you add, you're getting more specific).

### CODE REVIEW

File: src/CoreShop2VueStorefrontBundle/Bridge/ImporterFactory.php [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+        $this->hosts = $hosts;
+        $this->indexTemplate = $indexTemplate;
+        $this->stores = $stores;
+        $this->resolver = $this->configureOptions(new OptionsResolver());
+    }
+
+    /**
+     * @return array<ImporterFactory>
+     */
+    public function create(?string $store, ?string $language, ?string $type): array
```
Comment (kwreczycki) [inline src/CoreShop2VueStorefrontBundle/Bridge/ImporterFactory.php:66]: Can you add unit tests for this class ?

**Comment by kwreczycki** [comment]
Looks good, thank you for your input !!!

**Comment by dkarlovi** [comment]
Argh, rebased on wrong branch! :see_no_evil: sorry about that, I'll add the tests soon.

**Comment by kwreczycki** [comment]
Awesome, thanks

**Comment by dkarlovi** [comment]
@kwreczycki I guess you've closed this by accident, I've rebased to your master, but it's not showing up, maybe Github doesn't process closed PRs.

Thanks for having a look. :beers:

**Comment by kwreczycki** [comment]
Yep, sorry. Please open PR once again

**Pull request closed without merging**